.github/workflows/test_helm_charts.yaml

```diff
@@ -0,0 +1,46 @@
+name: "Validate Helm Charts downloads"
+
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/test_helm_charts.yaml"
+      - "nebari/**"
+  push:
+    branches:
+      - main
+      - develop
+    paths:
+      - ".github/workflows/test_helm_charts.yaml"
+      - "nebari/**"
+
+jobs:
+  test-helm-charts:
+    name: "Helm Charts Validation"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout Infrastructure"
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.8"
+      - name: Install additional Python dependencies
+        run: |
+          pip install python-hcl2
+          pip install tqdm
```

Comment:
Should we include these in our dev dependencies? Theoretically, we could run this script locally?